Tools/TestWebKitAPI/Tests/WebKitCocoa/IndexedDBSuspendImminently.mm

```diff
@@ -39,60 +40,27 @@
 
 #if PLATFORM(IOS_FAMILY)
 
-static bool idbActivitiesStarted;
-
-@interface IndexedDBSuspendImminentlyMessageHandler : NSObject <WKScriptMessageHandler>
-@end
-
-@implementation IndexedDBSuspendImminentlyMessageHandler
-
-- (void)userContentController:(WKUserContentController *)userContentController didReceiveScriptMessage:(WKScriptMessage *)message
-{
-    idbActivitiesStarted = true;
-    receivedScriptMessage = true;
-    lastScriptMessage = message;
```

Comment:
Can we just do here: 
```
receivedScriptMessage = true;
scriptMessages.append(message);
```
And use `[getNextMessage() body]` below?
(There is a `getNextMessage` in DeprecatedGlobalValues.h)

Reply:
We could, but this is a non-global replacement for DeprecatedGlobalValues.  I think it would be better to replace all use of getNextMessage() with TestScriptMessageHandler